CHANGES.md

```diff
@@ -0,0 +1,224 @@
+# Changes to `pyrealm`
+
+This document provides a brief overview of the main changes to `pyrealm` at each of the
+released versions. More detail can be found at the GitHub release page for each version.
+
+## 1.0.0
+
+- Addition of a more pythonic re-implementation of the SPLASH v1.0 model with a more
+  flexible user interface and faster calculation.
+- Revision of the optimal chi calculation internals - better internal structure and
+  extensible framework, with option to constrain xi values to allow slow acclimation of
+  xi.
+- Updated implementation of the SubdailyPModel (renamed from FastSlowPModel) to use the
+  new optimal chi structure and allow it to be used with all optimal chi models,
+  including C4 photosynthesis.
+- Updated and revised unit testing.
+- Dropped support for Python 3.9 for first main release and to adopt more recent typing
+  syntax.
+
+## 0.10.1
+
+- Addition of draft extension of subdaily memory_effect function that allows for missing
+  data arising from non-estimable Jmax and Vcmax.
+
+## 0.10.0
+
+- Implementation of the Mengoli et al 2023 soil moisture penalty factor. The existing
```

Comment:
Minor point, you didn't link the reference here

Reply:
They are linked in the main docs though. I'll probably keep this file simple as it isn't primarily user facing